types/node/v6/tsconfig.json

```diff
@@ -2,7 +2,7 @@
     "compilerOptions": {
         "module": "commonjs",
         "lib": [
-            "es5"
+            "es6"
```

Comment:
why this change? I can remember about some complains that node typings don't work with lib es5, usually because of missing forward declarations. if es6 is used here this is not detected by tests